.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!--
+
+If you are a [PowerShell Team](https://github.com/orgs/PowerShell/people) member, please make sure you choose the Reviewer(s) and Assignee for your PR.
+If you are not, feel free to choose them if you are familiar with the team. Alternatively, leave the fields blank and let maintainers choose them for you.
```

Comment:
I would switch this sentence around as the "familiar" path is less likely. So something like:

> If you are not part of the PowerShell Team, you can leave the fields blank and the Maintainers will choose them for you. If you are familiar with the team, feel free to add some Reviewers yourself.